### ISSUE TITLE
Regression in import order check

### ISSUE THREAD

**Issue by tiran** [PyCQA/flake8-import-order#87]
There seems to be a regression between 0.6.1 and 0.9.2. The following imports are no longer considered in wrong order (H, then C, the P). flake8-import-order == 0.6.1 reported this.

```
from custodia.plugin import HTTPConsumer, HTTPError
from custodia.plugin import CSStoreError, CSStoreExists, PluginOption
```
https://github.com/latchset/custodia/issues/73
https://github.com/latchset/custodia/pull/71

**Comment by pgjones** [comment]
Thanks, could you test https://github.com/pgjones/flake8-import-order/commit/64218d6c8c4ae51b48ef85493beee110ac9161c8 to see if it fixes this issue? I also need to check what other styles should do in this situation.

**Comment by Fantomas42** [comment]
@pgjones I have tested the patch, and it fixes the issue.

Best regards

**Comment by pgjones** [comment]
Thanks, fixed in 863f4597ab4e0879ee6058fab3f5e5b1df1f3463. Will release soon
